test/attr/attr_final.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 4
```

Comment:
`-swift-version 4`. You only get to add that if *everything* in the file is also being tested in Compatibility. (Right, @slavapestov?)